.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Clairvoyance
+
+**Clairvoyance** is an open-source project. We appreciate your contributions!
+
+## Issues
+
+If you have encountered an bug or want to propose your idea, you can open a new issue on the github.
+
+### Best practices
+1. Check for existing issues before opening a new one. If you find a similar one, add more details to it instead of creating a duplicate.
+2. Use relevant template and follow it.
+3. Add relevant details, it helps us in triaging and further steps.
+
+## Code contribution
+Pull requests are welcome! For major changes, please open an issue first to discuss what you would like to change.
+
+### Developer Environment Setup
+We recommend to use [VSCodium](https://vscodium.com/#install)/VSCode IDE with recommended extensions (automatically suggested).
+
+To resolve dependencies, execute the following commands in your terminal:
+```shell
+python -m pip install pipx
+pipx ensurepath
+pipx install poetry
+poetry config virtualenvs.in-project true
+poetry install
+```
```

Comment:
```suggestion
### Developer Environment Setup

To install dependencies, execute the following commands in your terminal:

```shell
python -m pip install pipx
pipx ensurepath
pipx install poetry
poetry config virtualenvs.in-project true
poetry install
```

We use [VSCodium](https://vscodium.com/#install) and VSCode IDEs with extensions found in the [`.vscode/extensions.json` file](../.vscode/extensions.json). Having said that, you might use other tools as long as the outcome follows the guidelines of this project.
```

Reply:
I'd leave more freedom to choose tools

Reply:
The description could be improved a bit: extensions are going to be automatically recommended upon project opening, like this:
![image](https://github.com/nikitastupin/clairvoyance/assets/39376984/8ef88e85-e67f-4983-96e5-019e03e8b3ef)

But I get the idea :)